test/v2/client/test_client_setup.go

```diff
@@ -24,6 +23,13 @@ var (
 
 const (
 	PreprodEnv = "../config/environment/preprod.json"
+
+	//G1URL         = "https://eigenda.s3.amazonaws.com/srs/g1.point"
+	//G2URL         = "https://eigenda.s3.amazonaws.com/srs/g2.point"
+	//G2PowerOf2URL = "https://eigenda.s3.amazonaws.com/srs/g2.point.powerOf2"
+	G1URL         = "https://srs-mainnet.s3.amazonaws.com/kzg/g1.point"
```

Comment:
Should we make `https://srs-mainnet.s3.amazonaws.com` a configuration value?

Reply:
I'm not sure this makes sense as a config value. The purpose of these is to download the SRS files on a developer laptop the first time they run the test suite. Once those files are present, these constants aren't used for anything. So in kubernetes where we are using a persistent volume, these files are ignored.

Reply:
Sounds good, I don't think this URL will change anyway.